lib/knackhq/client.rb

```diff
@@ -48,10 +48,9 @@ def records_by_page(object, page_number)
                      .records
                      .get(:params => { :page => page_number,
                                        :rows_per_page => 500 })
-                     .to_h
-      return [] if hash_request.empty?
-      transform_hash_keys = hash_request[:records]
-      symbolize_hash_keys!(transform_hash_keys)
+      payload = payload_hash(hash_request)
+      return [] if payload.empty?
+      payload[:records]
     end
 
     def records_info(object)
```

Comment:
Sorry just saw the comment. It returns a hash with the records information. For example, a record might have multiple fields, page numbers, etc.. I wanted to make sure that what was returned was an array (as is standard within the rest of our app).